birdhouse/env.local.example

```diff
@@ -142,12 +142,33 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 # "Notebook" are all the tutorial notebooks on Jupyter.
 #export AUTODEPLOY_NOTEBOOK_FREQUENCY="@every 5m"
 
+# Log directory used for extra autodeploy tasks
+# export AUTO_DEPLOY_LOGDIR=/var/log/PAVICS
```

Comment:
This location is not specific to autodeploy.  All kinds of jobs log there so all PAVICS related stuff are at one place.

I suggest naming this PAVICS_LOGDIR instead and the default value should be in `default.env`.